### DISCUSSION TITLE
Please add support for Polish Zloty (PLN) for sales

### DISCUSSION THREAD

**Discussion by marioosh** [gameserverapp/Platform#297] [Suggestions]
I'm building a cluster for polish community and they prefer to pay with local currency 
Polish Zloty (PLN) - which is already supported by many payment gateways including paypal.
Please add this option to for sales if possible.

**Comment by mvaessen** [comment]
Thanks for your suggestions!

The Zloty will be added to GSA in the upcoming Stripe update
